wp-cli.php

```diff
@@ -500,6 +588,59 @@ public function ip_remove( $args, $assoc_args ) {
 		);
 	}
 
+	/**
+	 * Sets list of IPs to whitelist. Overwrites current settings.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <ip>
+	 * : IP address to update.
+	 *
+	 * [--new-ip]
+	 * : The IP address to replace with.
+	 *
+	 * [--new-label]
+	 * : The new label for the IP address.
+	 *
+	 * [--network]
+	 * : Multisite only. Sets configuration for the network as a whole.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *    # Sets IP whitelist to 192.0.0.1.
+	 *    $ wp rsa ip-update 192.0.0.1
+	 *    Success: Updated site IP whitelist to 192.0.0.1.
+	 *
+	 * @subcommand ip-update
+	 *
+	 * @param array $args       List of IPs to set.
+	 * @param array $assoc_args Optional flags.
+	 */
+	public function ip_update( $args, $assoc_args ) {
+		$this->setup( $args, $assoc_args );
+
+		if ( 0 === count( $assoc_args ) ) {
+			\WP_CLI::warning( __( 'Provide the arguments to update.' ), 'restricted-site-access' );
```

Comment:
Because this command supports single IP only, the associate arguments are required. If no flag was provided, the command will do nothing. I think we should raise an error here instead of a warning.